.github/workflows/python-app.yml

```diff
@@ -0,0 +1,27 @@
+name: Python package
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pytest
+        pip install -r will/requirements/dev.txt
+        pip install tox tox-gh-actions
```

Comment:
tox is already included in `dev.txt`

Reply:
Yep that’s easy enough.

Reply:
I also removed the extra pip install pytest.